src/Chapter1.hs

```diff
@@ -632,7 +644,11 @@ Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
 
-sumLast2 n = error "sumLast2: Not implemented!"
+sumLast2 :: Int -> Int
+sumLast2 n = lastDigit' + penultimateDigit
+ where
+  lastDigit'       = abs n `mod` 10
+  penultimateDigit = (abs n `div` 10) `mod` 10
```

Comment:
That is a wonderful solution! 👏🏼  You correctly noticed that it is the `div` and `mod`, cool 😎 

One hint to make your solution even shorter, I see that you have both:

```haskell
abs n `mod` 10
abs n `div` 10
```
The standard library has the [`divMod`](https://hackage.haskell.org/package/base-4.14.0.0/docs/Prelude.html#v:divMod) function, that actually combines inside both `div` and `mod`. And this is exactly what you use!.

So you could write it this way:

```haskell
(x, y) = abs n `divMod` 10
```

You can see how we could pattern match on the pair 🙂

Reply:
Oh yes, thank you - `divMod`'s way better!